csvqb/csvqb/models/cube/csvqb/components/attribute.py

```diff
@@ -38,6 +43,44 @@ def new_attribute_values(self) -> List[NewQbAttributeValue]:
         pass
 
 
+@dataclass
+class QbAttributeLiteral(QbAttribute, ABC):
```

Comment:
I know none of the others here have a docstring - but it'd be good to add one here for the API documentation. So people can understand what the class is meant to represent.

Reply:
Resolved.